lib/src/trimmer.dart

```diff
@@ -277,7 +277,7 @@ class Trimmer {
       await videoPlayerController.pause();
       return false;
     } else {
-      if (videoPlayerController.value.position.inMilliseconds >=
+      if (videoPlayerController.value.position.inMilliseconds <=
           endValue.toInt()) {
         await videoPlayerController
             .seekTo(Duration(milliseconds: startValue.toInt()));
```

Comment:
Here, it checks if the video is currently in a paused state (but it has reached the end) then just seek it to the start position, and then start the playback (when this is called). So, I am checking if it is equal to the end position basically. 

But here I have also included greater than condition because sometimes the value becomes a bit greater than the end position (maybe in nanoseconds), so in order to address that condition, I have included this.